calculate_average.js

```diff
@@ -0,0 +1,14 @@
+
+const calculateAverage = function calculateAverage(grades) {
+
+  let total = 0;
+  for (let i = 0; i < grades.length; i++) {
```

Comment:
you could put grades.length into a variable to avoid recalculating it every loop